benchmarks/dynamo/common.py

```diff
@@ -2198,6 +2198,38 @@ def get_benchmark_indices(self, length):
         )
         return start, end
 
+    def get_fsdp_auto_wrap_policy(self, model_name: str) -> Optional[ModuleWrapPolicy]:
+        from diffusers.models.transformer_2d import Transformer2DModel
```

Comment:
just curious, are `diffusers` and `transformers` packages installed in dynamo runs? I assume so since CI is green but just double checking